ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -41,3 +68,12 @@ def get_colored_area(self, image, lower, upper):
 
         return area, extracted_image
 
+
+    def process_image(self, image):
+
+        desired_shape = (128, 128)
+        image = cv2.resize(image, desired_shape, cv2.INTER_LINEAR)
+        image = image.astype('float32') / 255
+        processed_image = image.reshape(1, *image.shape)
```

Comment:
Just to make sure - your network is fully connected and you're casting image to a vector? Nothing wrong about it, but want to confirm I understand why this reshaping ^^

Reply:
Ok, now I now you were reshaping to batch of size 1 :)